keyboards/ciaanh/kanagawa/info.json

```diff
@@ -0,0 +1,224 @@
+{
+    "keyboard_name": "Kanagawa",
+    "url": "https://github.com/Ciaanh/keyboards/tree/main/Kanagawa",
+    "manufacturer": "Ciaanh",
+    "maintainer": "Ciaanh",
+    "usb": {
+        "vid": "0xC1E0",
+        "pid": "0x0010",
+        "device_version": "2.1.0"
+    },
+    "bootmagic": {
+        "matrix": [0, 13]
+    },
+    "encoder": {
+        "rotary": [
+            {"pin_a": "GP2", "pin_b": "GP3"}
+        ]
+    },
+    "diode_direction": "COL2ROW",
+    "matrix_pins": {
+        "rows": ["GP28", "GP27", "GP26", "GP22", "GP21", "GP20"],
+        "cols": ["GP5", "GP6", "GP7", "GP8", "GP9", "GP10", "GP11", "GP12", "GP13", "GP14", "GP15", "GP16", "GP17", "GP18"]
+    },
+    "processor": "RP2040",
+    "bootloader": "rp2040",
+    
+    "layouts": {
+        "LAYOUT_ansi": {
```

Comment:
A single info.json with both ansi/iso layout macros is invalid given it would be incompatible. Each sub-revision should have its own info.json with only the relevant layout macro.